### ISSUE TITLE
Cypress "I should see" step definitions should use `be.visible`, not `exist`.

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#11609]
## Description
I can't remember why I changed these, but they should be changed back, additional step definitions added as necessary, etc.

## Acceptance Criteria
- [x] A new step definition has been created that checks _existence_ of elements with the specified selector, rather than _visibility_.
- [x] The existing `should (not) see` step definitions are altered to use _visibility_ rather than _existence_.
- [x] Also change `the image` and `the element` to `an image` and `an element`.
- [x] All tests use the appropriate step definitions.

**Comment by chri5tia** [comment]
https://github.com/department-of-veterans-affairs/va.gov-cms/pull/11611 Draft PR

**Comment by chri5tia** [comment]
Something to note for the future, might be resolved already: https://github.com/cypress-io/cypress/issues/9348

**Comment by EWashb** [comment]
@chri5tia is this in progress? I see it is assigned to you, but has the Sitewide team label

**Comment by ndouglas** [comment]
I can take it over since there hasn't been recent movement.

**Comment by chri5tia** [comment]
@ndouglas I was working on it, I just didn't push anything. I hope we didn't duplicate efforts... see my PR: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/12344

There are a few minor issues and a couple more things I need to do, but should be done today. Not sure how I ended up doing this with a Sitewide label, but I think I created this issue or talked to Nate about doing this?

**Comment by ndouglas** [comment]
Cool, thanks!

**Comment by jilladams** [comment]
I talked to Christia about this ticket today to understand how it ties to #12220 . For visibility: this ticket was created after this PR was merged in the fall: https://github.com/department-of-veterans-affairs/va.gov-cms/pull/11613. Christia had it top of mind as a mental backlog thing to close out, but: we didn't get it tagged correctly for sprint purposes. 

So: PW has noted: we'll make sure assignment and team tags get added for future things that have crossover with CMS team.
